docs/README.md

```diff
@@ -4,6 +4,8 @@
 
 This is a payment authorization app for the Affirm payment method (financing with monthly payments).
 
+NOTE: Currently we do not support a marketplace situation, where the seller processes the payment

```

Comment:
```suggestion
> ⚠️ _Currently Affirm cannot be used in a marketplace scenario where the seller processes the payment. In marketplace scenarios, the marketplace itself must process the payment._
```

Reply:
This has been addressed in the following commit.